webui/ocw/apps.py

```diff
@@ -1,5 +1,45 @@
 from django.apps import AppConfig
+import os
+import logging
+from apscheduler.schedulers.background import BackgroundScheduler
+from apscheduler.executors.pool import ThreadPoolExecutor
+from pytz import utc
+
+logger = logging.getLogger(__name__)
+__scheduler = None
+
+
+def getScheduler():
+    global __scheduler
+    if __scheduler is None:
+        logger.debug("Create new BackgrounScheduler")
+        executors = {
+                'default': ThreadPoolExecutor(1),
+            }
+        job_defaults = {
+                'coalesce': False,
+                'max_instances': 1
+            }
+        __scheduler = BackgroundScheduler(executors=executors, job_defaults=job_defaults, timezone=utc)
+    return __scheduler
 
 
 class OcwConfig(AppConfig):
     name = 'ocw'
+    ready_called = False
+
+    def ready(self):
+        if os.environ.get('RUN_MAIN', None) != 'true' or self.ready_called:
+            return
+        self.ready_called = True
+
+        getScheduler().start()
+
+        try:
+            import ocw.lib.db
+            import ocw.lib.cleanup
+
+            ocw.lib.db.init_cron()
```

Comment:
don't you think that it would be more clear just to directly do 
`getScheduler().add_job(cleanup_run, trigger='interval', minutes=60, id='cleanup_all')` here ?

Reply:
this comment is actually for next line but I would assume idea would be the same for this line too

Reply:
The thing is that the "id", at least for the db updater, is meaning full. And then I would need to share this "name" in two files.
And the second one, is just because it might be needed in the future...